.github/ISSUE_TEMPLATE/tracking.md

```diff
@@ -0,0 +1,61 @@
+---
+name: "📊 Tracking Issue"
+title: "📊Tracking: [service]"
+about: Add a module tracking issue (internal use only)
+labels: management/tracking
+---
+
+### Add your +1 👍  to help us prioritize high-level constructs for this service
+--- 
+
+### Overview:
+<!-- 
+Summary of the service (leverage the service’s product page for the text) and a link to the relevant AWS Docs. 
+This should be the same text that we put at the top of the package’s README.md. Also include a link to the 
+service’s CDK Construct Library API reference page.
+-->
+
+
+
+
+
+
+[AWS Docs](url)  <!-- replace `url` with link to the relevant AWS Docs -->
+[CDK API Reference](url) <!-- replace `url` with link to the service's CDK API reference -->
+
+
+### Implementation:
+<!-- 
+Checklist of use cases, constructs, features (such as grant methods) that will ship in this package 
+(not required until the issue is added to the public roadmap)
```

Comment:
Just want to point out that the checklist does not need to be 100% complete when added to the roadmap.  This will be living issue, and the checklist will be updated throughout the 'experimental' phase.